inc/functions_post.php

```diff
@@ -1021,3 +1021,30 @@ function get_post_attachments($id, &$post)
 		}
 	}
 }
+
+/**
+ * Returns bytes count from human readable string
+ * Used to parse ini_get human-readable values to int
+ *
+ * @param string $val Human-readable value
+ */
+function return_bytes($val) {
+	$val = trim($val);
+	if ($val == "")
+	{
+		return 0;
+	}
+
+	$last = strtolower($val[strlen($val)-1]);
```

Comment:
These should probably be `my_strtolower` and `my_strlen` to handle any possible multibyte strings.

Reply:
I do not think so... This is only useful to parse possible values of ini_get, and they are [0-9]+[kmg] - no use for user-specified input, thus no place for multibyte strings

Reply:
Ah, fair enough. Should be fine as is then!